pkg/repocache/cache.go

```diff
@@ -85,7 +88,48 @@ func (rp *GitRepoCache) GetEntry(url git_url.GitUrl) (*CacheEntry, error) {
 	rp.reposMutex.Lock()
 	defer rp.reposMutex.Unlock()
 
-	e, ok := rp.repos[url.NormalizedRepoKey()]
+	urlN := url.NormalizedRepoKey()
+
+	// evaluate overrides
+	for _, ro := range rp.repoOverrides {
+		uO, err := git_url.Parse(ro.RepoKey)
+		if err != nil {
+			return nil, err
+		}
+		uN := uO.NormalizedRepoKey()
+
+		// FIXME, what was the requested logic here?
```

Comment:
can you update the requested logic herE?

Reply:
I'll try to checkout your branch and add a commit that you can then cherry-pick.